src/video-player.js

```diff
@@ -125,6 +126,15 @@ class VideoPlayer extends Utils.mixin(Eventable) {
 
   _setVideoJsListeners(ready) {
 
+    // Prevent flash of error message while lazy-loading plugins.
+    videojs.hook('beforeerror', (player, err) => {
+      if (err && err.code === 3 && this.loadingLazyPlugins) {
```

Comment:
`code == 3` means a media issue i.e. corrupted manifest etc
I used it like this because I saw it being used this way in a few other places, I'm in for refactoring this, but I don't think this is the place to do that.